cmake/toolchain.cmake

```diff
@@ -15,3 +15,17 @@ set(CMAKE_CXX_COMPILER_WORKS TRUE)
 
 # Project setup
 enable_language(C CXX ASM)
+
+# Clear FLAGS for all languages
+# Mbed OS uses different init flags (optimization, debug symbols) 
+set(CMAKE_C_FLAGS_DEBUG "" CACHE STRING "" FORCE)
```

Comment:
For reviewers, as our profile flags differ (android for instance does the same in their toolchain file) we clear the the flags after init (happens on line 17th in enable_language). Just making sure we have flags we set.